internal/manifest/version.go

```diff
@@ -453,6 +450,18 @@ func (m *FileMetadata) LatestRef() {
 	}
 }
 
+// VirtualSpaceAmpProportion computes a virtual sstable's contribution to space
+// amplification. See comment above FileBacking.VirtualizedSize for the
+// reasoning behind this computation.
+func (m *FileMetadata) VirtualSpaceAmpProportion() uint64 {
+	if !m.Virtual {
+		return 0
+	}
+	virtualizedSize := m.FileBacking.VirtualizedSize.Load()
+	latestRefs := m.LatestRefs()
+	return (m.FileBacking.Size - virtualizedSize) / uint64(latestRefs)
```

Comment:
I wonder if we should be dividing by `m.Size` instead of `latestRefs` here - that way it's easier to reason about smaller virtual sstables getting a greater size adjustment than bigger virtual sstables that reference the same underlying sstable. Thoughts?